tasks/nms/install-redhat.yml

```diff
@@ -33,14 +33,33 @@
   register: nms_install_state
   when: nms_cleanup_status is not defined
 
+- name: (Amazon Linux/CentOS/Oracle Linux/RHEL) NMS Modules
+  ansible.builtin.yum:
+    name: "{{ item.name }}-{{ '*' if (item.version is not defined or item.version == '') else item.version }}"
```

Comment:
Same here as [above comment](https://github.com/nginxinc/ansible-role-nginx-management-suite/pull/13/files#r1140714747). I'd leave off `-*` when we are installing/upgrading

Reply:
Thanks @jswongF5, this has been addressed in the latest changes